packages/@vuepress/shared-utils/src/getPermalink.ts

```diff
@@ -33,7 +64,7 @@ export = function getPermalink ({
   }
   slug = encodeURI(slug)
 
-  const d = new Date(date)
+  const d = new Date(addTzOffset(toUtcTime(date)))
```

Comment:
I guess `new Date()` is  redundant.

Reply:
Yes, I suppose you're right since `addTzOffset` always returns a `Date` instance.